features/vendor_bids_too_high.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Vendor bids too high
```

Comment:
should we include this spec in the "vendor bids" feature spec file? Not sure how big that is, but I do think we cover a few different cases in there already so this might fit nicely 

https://github.com/18F/micropurchase/blob/develop/features/vendor_bids_reverse_auction.feature
https://github.com/18F/micropurchase/blob/develop/features/vendor_bids_sealed_auction.feature